tests/benchmark/snappy/dynamic/PerfRun_Jdbc.conf

```diff
@@ -0,0 +1,102 @@
+#Do not change the way of queries, sparkProperties, sparkSqlProperties. Just change the values inside strings
```

Comment:
Before running the perf test, we create a PerfRun_Jdbc.conf from the PerfRun_Jdbc.conf.template, so this file can be deleted or renamed to PerfRun_Jdbc.conf.template